prow/config.yaml

```diff
@@ -1104,6 +1104,73 @@ presubmits:
           path: /mnt/disks/ssd0/docker-graph
       - name: var-lib-docker
         emptyDir: {}
+  - name: pull-kubernetes-e2e-containerd-gce
+    agent: kubernetes
+    context: pull-kubernetes-e2e-containerd-gce
+    rerun_command: "/test pull-kubernetes-e2e-containerd-gce"
+    trigger: "(?m)^/test pull-kubernetes-e2e-containerd-gce,?(\\s+|$)"
+    always_run: false
+    skip_report: false
+    branches:
+    - master
+    spec:
+      containers:
+      - args:
+        - --root=/go/src
+        - --repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)
+        - --repo=k8s.io/release
+        - --repo=github.com/kubernetes-incubator/cri-containerd
+        - --upload=gs://kubernetes-jenkins/pr-logs
+        - --git-cache=/root/.cache/git
+        - --clean
+        - --timeout=90
+        # the release-1.6 version of this job uses Jenkins, so here we override
```

Comment:
nit: since this job doesn't need or have a 1.6 version on Jenkins, this comment should be removed and these flags should go with the rest of the job config for now.

Reply:
Will do.

Reply:
Done. Removed the comments.